pyproject.toml

```diff
@@ -62,17 +62,17 @@ documentation = [
 development = [
     "build>=1.2.1",
     "codespell>=2.3.0",
-    "coverage>=7.5.3",
-    "mypy>=1.10.0",
-    "pylint>=3.2.3",
+    "coverage>=7.6.0",
```

Comment:
Bump 3party versions needs to be in a seperate PR; only with the bump.

Reply:
Moved to #2241.